tdrs-frontend/src/components/Reports/Reports.jsx

```diff
@@ -125,6 +125,24 @@ function Reports() {
     setTouched((currentForm) => ({ ...currentForm, stt: true }))
   }
 
+  const constructYearOptions = () => {
+    const years = []
+    const today = new Date(Date.now())
+
+    const fiscalYear =
+      today.getMonth() > 8 ? today.getFullYear() + 1 : today.getFullYear()
+
+    for (let i = 2020; i <= fiscalYear; i++) {
```

Comment:
For each year from 2020 until the current Fiscal year an option element will be created.

Reply:
Use descending order

Reply:
oldest yeah should be 2021